packages/twenty-website/src/database/init-database.ts

```diff
@@ -0,0 +1,50 @@
+import { global } from '@apollo/client/utilities/globals';
+import { graphql } from '@octokit/graphql';
+
+import { fetchAndSaveGithubStars } from '@/app/contributors/api/fetch-and-save-github-stars';
+import { fetchAssignableUsers } from '@/app/contributors/api/fetch-assignable-users';
+import { fetchIssuesPRs } from '@/app/contributors/api/fetch-issues-prs';
+import { saveIssuesToDB } from '@/app/contributors/api/save-issues-to-db';
+import { savePRsToDB } from '@/app/contributors/api/save-prs-to-db';
+import { IssueNode, PullRequestNode } from '@/app/contributors/api/types';
+import { migrate } from '@/database/database';
+
+export const initDatabase = async () => {
+  if (!global.process.env.GITHUB_TOKEN) {
+    return new Error('No GitHub token provided');
+  }
+
+  console.log('Synching data..');
+
+  const query = graphql.defaults({
```

Comment:
This code is related to Github, shouldn't be in the database folder. What you should do instead is isolate the logic into a folder dedicated to the Github sync. Also make sure you don't copy/paste too much code, this seems very similar to existing code in init/route.tsx?